src/main/java/com/minecolonies/coremod/commands/ActionArgumentType.java

```diff
@@ -170,11 +201,32 @@ public int allowedSpaceCount()
         }
     }
 
-    private List<String> getCitizenTabCompletions(final String potentialArgumentValue)
+    private List<String> getCitizenTabCompletions(@NotNull final TreeNode<IMenu> actionMenuTreeNode, final String potentialArgumentValue)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The Cyclomatic Complexity of this method "getCitizenTabCompletions" is 18 which is greater than 15 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this method to reduce its Cognitive Complexity from 44 to the 20 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS3776)